service/core/client/src/main/resources/reference.conf

```diff
@@ -20,6 +20,12 @@ lagom.circuit-breaker {
     # Duration of time in open state after which to attempt to close
     # the circuit, by first entering the half-open state.
     reset-timeout = 15s
+
+    # A whitelist of fqcn of Esxceptions that the CircuitBreaker
+    # should not consider failures. By default all exceptions are
+    # failures
+    #
+    exception-whitelist = []
```

Comment:
Users would be able to whitelist Lagom or their own exceptions which, under their criteria, should not be counted by the CircuitBreaker as failures. The default is an empty list meaning all exceptions are considered failures which is the current behaviour.